extensions/2.0/Vendor/EXT_feature_metadata/schema/featureCollection.property.schema.json

```diff
@@ -0,0 +1,30 @@
+{
+  "$schema": "http://json-schema.org/draft-04/schema",
+  "title": "Property array",
+  "type": "object",
+  "description": "Array containing property data",
+  "allOf": [ { "$ref": "glTFProperty.schema.json" } ],
+  "properties": {
+      "elementCount": {
+          "type": "integer",
+          "minimum": 1,
+          "description": "The number of elements in the array. `elementCount` may be greater than `featureCount` if the array contains extra elements not used by this model, e.g. if different models refer to different sections of a global array or if the arrays represents a larger data dictionary."
+      },
+      "elementByteLength": {
+          "type": "integer",
+          "minimum": 1,
+          "description": "A fixed-size byte length for `STRING` and `BINARY` elements. When omitted, elements are variable-size."
+      },
+      "bufferView": {
+          "allOf": [ { "$ref": "glTFid.schema.json" } ],
+          "description": "The index of the `bufferView` containing property array data. TODO: need to define binary packing rules, including padding rules."
+      },
+      "offsetsBufferView": {
+          "allOf": [ { "$ref": "glTFid.schema.json" } ],
+          "description": "The index of the `bufferView` containing offsets to elements. Required for variable-size strings, variable-size binary elements, and variable-size arrays. The offsets buffer contains `elementCount + 1` unsigned integers (TODO: 32-bit or 64-bit, or configurable), which represent the start position of each element in the buffer, with the last offset representing the position after the last element. The string byte length, binary value byte length, or element array length is computed using the difference between the current offset and the subsequent offset. TODO: confusing that this applies to both arrays and string/binary types."
```

Comment:
I don't think this is too confusing as long as the following points are clear

* string features are just a special case of variable size arrays where the elements are UTF-8 characters (which are an exception for being between 1-4 bytes)
* binary data is just a variable-size array of `UINT8` (see my other comment). then

Therefore all three cases are still essentially a variable-length array.